grommet/hpe-design-system
# take out medium box height

[Deploy Preview](https://deploy-preview-2700--keen-mayer-a86c8b.netlify.app/templates/filtering?q=fil#filtering-with-selectable-results)

#### What does this PR do?
This PR takes out the `height` set in the `box` containing the DataTable.
#### Where should the reviewer start?
FilterServers.js
#### What testing has been done on this PR?
site testing
In addition to the feature you are implementing, have you checked the following:

**General UX Checks**

**Accessibility Checks**

**Code Quality Checks**

#### How should this be manually tested?
filter through data and make sure it still has the correct spacing or use the search
#### Any background context you want to provide?
Im not sure why the `height` was set but since it was the filter data was trying to fill that height which was making it not have the correct spacing 
#### What are the relevant issues?
closes #2663 
#### Screenshots (if appropriate)

#### Should this PR be mentioned in Design System updates?
no
#### Is this change backwards compatible or is it a breaking change?
backwards compatible
